### PR TITLE
Add new button for generating http connections and components

### PR DESCRIPTION
**Pull request by hkhattabii**
**Styles**
The button and the tooltip will be made from the **material-ui library**

**Button position**

The button will be placed on the left of the "create script" button with a small spacing and aligned with it. 
 
**On mouse hover**

When the mouse hover the button, a **tooltip** appears to provide more information about the button.
This **tooltip** will be displayed on the top of the button. It will have black background color and white text color.
The **tooltip** appear with a transition animation that can be user-friendly.

**On mouse cursor leave**

When the mouse cursor leave the button, the tooltip disappears with also a transition animation.

**On mouse click**
When the button is triggered, a new page will appear that display the empty information of the documentation transformation.

**UI Vizualisation**

Index

### CODE REVIEW

File: src/views/design/ScriptsOverview/index.tsx [hunk 1]
```diff
@@ -195,17 +203,46 @@ const ScriptsOverview = withStyles(styles)(
...
-                                                    size="small"
-                                                    startIcon={<AddRounded />}
-                                                    onClick={() => redirectTo({ routeKey: ROUTE_KEYS.R_SCRIPT_NEW })}
-                                                >
-                                                    <Translate msg="scripts.overview.header.add_button" />
-                                                </Button>
+                                            <Box display="flex" alignItems="center">
+                                                <Box flex="0 0 auto" mr="8px" width="220px">
+                                                    <Tooltip
+                                                        title="Transform an OpenAPI documentation into IESI concept"
```
Comment (hkhattabii) [inline src/views/design/ScriptsOverview/index.tsx:209]: I just have seen how to render the translation without rendering the Translate component, I will change this line later

File: src/views/translations/en_GB.yml [hunk 2]
```diff
@@ -80,6 +80,8 @@ scripts:
     header:
       amount: 'You have {amount} scripts'
       add_button: Create new script
+      generate_button: "Generate connections and components"
+      transform_openapi_tooltip: "Transform an OpenAPI documentation into IESI concept"
```
Comment (BerrevoetsRobbe) [inline src/views/translations/en_GB.yml:84]: "Transform an OpenAPI documentation into IESI connections and components"

File: src/views/translations/en_GB.yml [hunk 3]
```diff
@@ -80,6 +80,8 @@ scripts:
     header:
       amount: 'You have {amount} scripts'
       add_button: Create new script
+      generate_button: "Generate connections and components"
```
Comment (BerrevoetsRobbe) [inline src/views/translations/en_GB.yml:83]: Load OpenAPI documentation

File: src/views/design/ScriptsOverview/index.tsx [hunk 4]
```diff
@@ -195,17 +203,50 @@ const ScriptsOverview = withStyles(styles)(
...
+                                                            onClick={() => {
+                                                                redirectTo({ routeKey: ROUTE_KEYS.R_OPENAPI });
+                                                            }}
+                                                        >
+                                                            <Translate
+                                                                msg="scripts.overview.header.generate_button"
+                                                            />
+                                                        </Button>
+                                                    </Tooltip>
+                                                </Box>
```
Comment (BerrevoetsRobbe) [inline src/views/design/ScriptsOverview/index.tsx:234]: make sure text is visible on 1 line

File: src/views/design/ScriptsOverview/index.tsx [hunk 5]
```diff
@@ -195,17 +203,50 @@ const ScriptsOverview = withStyles(styles)(
                                     </Box>
                                     {checkAuthorityGeneral(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE)
```
Comment (BerrevoetsRobbe) [inline src/views/design/ScriptsOverview/index.tsx:204]: the 'load OpenAPI' button should appear if you have CONNECTIONS_WRITE or COMPONENTS_WRITE privilege.
The 'add button " button should appear if you have SCRIPTS_WRITE privilige (as is situation)

**Review by BerrevoetsRobbe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**